guides/overview/examples.md

```diff
@@ -35,6 +35,7 @@ list of interesting projects that may be useful to others.
 - [ansible-statuscake](https://github.com/labynocle/ansible-statuscake)
 - [statuscake-push-client](https://github.com/vparpoil/status-cake-push-client)
 - [statuscake-exporter](https://github.com/chelnak/status-cake-exporter)
+- [status-cake-exporter](https://github.com/chelnak/status-cake-exporter)
```

Comment:
Did you perhaps mean to edit the previous line? These both point to the same project. It seems only the name is different, which would be my fault for incorrectly adding it in the first place.

Reply:
Oh dear. You are right! I didn't notice this at the time.

Yes maybe this can be closed in favour of a simpler commit that corrects the name above!

Reply:
Sure thing. I'll close this and await another PR.